scripts/setup-seafile.sh

```diff
@@ -330,6 +330,7 @@ function gen_seafdav_conf () {
 enabled = false
 port = 8080
 fastcgi = false
+host = localhost
```

Comment:
Since `fastcgi`'s default value is false, the default value of `host` should be `0.0.0.0`.